Docker/get-versions.ps1

```diff
@@ -57,12 +65,17 @@ function Invoke-SemanticSort {
         $Versions
     )
 
+    $major = @{label="major";expression={[int]$_.Split(".")[0]}}
+    $minor = @{label="minor";expression={[int]$_.Split(".")[1]}}
+    $patch = @{label="patch";expression={[int]$_.Split(".")[2]}}
+
     $Versions `
-        | Select-Object {$_.Split(".")} `
-        | Sort-Object {$_.'$_.Split(".")'[0], $_.'$_.Split(".")'[1], $_.'$_.Split(".")'[2]} -Descending `
-        | ForEach-Object { $_.'$_.Split(".")' -Join "." }
+        | Select-Object $major, $minor, $patch `
+        | Sort-Object major, minor, patch -Descending
```

Comment:
A `` ` `` must be added at the end of this line to avoid a syntax error.

Reply:
That's strange - it sure looks like a syntax error, yet I've run this many times without a problem, on my localhost and on a VM. I'm running PowerShell 7; maybe it accepts the pipe without needing a backtick to continue on the next line.